src/PlusImageProcessing/Testing/vtkPlusTransverseProcessEnhancerTest.cxx

```diff
@@ -36,10 +62,130 @@ int main(int argc, char **argv)
 
   vtkPlusLogger::Instance()->SetLogLevel(verboseLevel);
 
-  ///////////////
+  //check for various non-optional command line arguments
+  int commandCheckStatus = NULL;
+
+  if (inputFileName.empty())
+  {
+    std::cerr << "--input-seq-file not found!" << std::endl;
+    commandCheckStatus = EXIT_FAILURE;
+  }
+  if (outputFileName.empty())
+  {
+    std::cerr << "--output-seq-file not found!" << std::endl;
+    commandCheckStatus = EXIT_FAILURE;
+  }
+  if (configFileName.empty())
+  {
+    std::cerr << "--config-file not found!" << std::endl;

```

Comment:
Use LOG_ERROR. It prints to cerr but it also logs the error and we can make sure the message is consistently formatted, reference is included to filename, line number, etc.